bundles/azure-bundle/src/main/java/org/apache/gravitino/abs/fs/ABSFileSystemProvider.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.gravitino.abs.fs;
+
+import com.google.common.collect.ImmutableMap;
+import java.io.IOException;
+import java.util.Map;
+import javax.annotation.Nonnull;
+import org.apache.gravitino.catalog.hadoop.fs.FileSystemProvider;
+import org.apache.gravitino.catalog.hadoop.fs.FileSystemUtils;
+import org.apache.gravitino.storage.ABSProperties;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+
+public class ABSFileSystemProvider implements FileSystemProvider {
+
+  private static final String ABS_PROVIDER_SCHEME = "wasbs";
```

Comment:
What's the difference between wasb and wasbs?

Reply:
Path with prefix `wasbs` will use SSL in the transport process and `wasb` will not. It's advisable to use `wasbs` in the production environment.

Reply:
Azure provides several storages, like wasb, adls2, can you please investigate more.

Reply:
In the early days,  Azure only supported block storage and the corresponding protocol was `wsab` or `wsabs`. As time went by, azure added the support for ADLS, which supports directory management like HDFS to support big data eco-systems and the protocol is `abfs` or `abfss`(`abfss` is the securable enhancement of `abfs`).

Another point is that the version of Hadoop we use is `3.1` and only support `wsab`, we need to upgrade to `3.3`to
support this feature. I have created an issue about version upgrades. 

https://github.com/apache/gravitino/issues/5532

@jerryshao Please help to verify whether we need to update the Hadoop version to 3.3 or only use 3.3 for `hadoop-azure` (I'm afraid there will be some problem if we use Hadoop common with version 3.1 and hadoop-azure 3.3).

Reply:
@yuqi1129 As far as I know, hadoop environment using 3.1 and hadoop-azure using 3.2.1 can run normally (abfs/abfss protocol was first supported in hadoop-azure 3.2.0). However, whether hadoop-azure 3.3 is feasible needs further testing.

Reply:
I see, I'm not completely confident that Hadoop Common 3.1 is compatible with Hadoop-azure 3.2. 
Compared to using Hadoop 3.2 or above for both two, would you suggest trying Hadoop 3.1 and Hadoop-azure 3.2?

Reply:
I have no particular preference for this.
In terms of compatibility, our current production environment is using Hadoop 3.1 and Hadoop-Azure 3.2.1, which have been running for more than a year, and no compatibility issues have been found.
If we consider upgrading the Hadoop version, can you consider upgrading the Hadoop version of the entire project to a relatively stable version during this upgrade process? Currently, Hive still uses Hadoop 2.7. If we upgrade to a higher version, it may cause greater differences in the future.